src/components/Organisms/TodoCard/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+import AddTaskButton from "../../Atoms/AddTaskButton/index";
+import Task from "../../Molecules/Task/index";
+import COLOR from "../../../variables/color";
+
+const TodoCard = () => {
+  const [todoTask, setTodoTask] = useState([]);
+  const [taskAddOREdit, setAddOREdit] = useState(false);
+
+  let todoTaskList = [];
+  let todoStateTask = [];
+
+  const handleTaskComplete = (id) => {
+    let completedTask = todoTask.map((task) => {
+      if (task.id === id) {
+        task.state = "DONE";
+      }
+      return task;
+    });
+    setTodoTask(completedTask);
+  };
+
+  const handleEditComplete = (id, taskName) => {
+    let edittedTask = [];
+    if (taskName === "") {
+      edittedTask = todoTask.filter((task) => {
+        return task.id !== id;
+      });
+    } else {
+      edittedTask = todoTask.map((task) => {
+        if (task.id === id) {
+          task.name = taskName;
+        }
+        return task;
+      });
+    }
+    setTodoTask(edittedTask);
+    setAddOREdit(false);
+  };
+
+  // task.stateがTODOのものだけ抽出する
+  todoStateTask = todoTask.filter((task) => {
+    return task.state === "TODO";
+  });
+
+  todoTaskList = todoStateTask.map((task, index) => {
+    // addTask()が実行されたとき、todoTaskの一番後ろのコンポーネントだけにdefaultIsEditingを渡す
+    return (
+      <StyledTodoListItem key={index}>
+        <Task
+          key={task.id}
+          defalutValue={task.name}
+          defalutIsEditing={
+            taskAddOREdit ? task.id === todoTask.length - 1 : false
+          }

```

Comment:
```suggestion
          defalutIsEditing={true}
```
手元で試した限りこれで問題なさそうです。
（初回レンダリング（マウント時）にのみ関係があるpropsなので、親コンポーネントの方で状態管理する必要がない気がします。）

Reply:
それと、　defalut -> default だと思います。